app/views/unattended/provisioning_templates/provision/windows_default_provisioning.erb

```diff
@@ -12,13 +12,15 @@ description: |
   - systemLocale: en-US
   - systemUILanguage: en-US
   - systemTimeZone: GMT Standard Time #see https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
-  - wimImageName = Windows 8.1 Pro # Image name seems only necessary if the WIM contains more than one image
+  - wimImageName = Windows 10 Pro # Image name seems only necessary if the WIM contains more than one image
 -%>
 <%
   system_locale = host_param('systemLocale') ? host_param('systemLocale') : 'en-US'
   system_ui_lang = host_param('systemUILanguage') ? host_param('systemUILanguage') : 'en-US'
   setup_ui_lang = 'en-US'
+  input_locale = host_param('inputLocale') ? host_param('inputLocale'): 'de-DE'
```

Comment:
Do you really want to set the german keyboard-layout as the default?
I would use the en-US one as default for the foreman-community

Reply:
True, thanks!